launcher/src/main/scala/bloop/launcher/BloopServerVersion.scala

```diff
@@ -12,13 +12,13 @@ case class BloopServerVersion(
 object BloopServerVersion {
   def apply(serverVersion: String, out: PrintStream): Option[BloopServerVersion] = {
     Version(serverVersion).items.toList match {
-      case (major: Version.Number) :: (minor: Version.Number) :: rest =>
+      case Version.Number(major) :: Version.Number(minor) :: rest =>
```

Comment:
@alexarchambault This is a revert of the changes I did for bloop to upgrade to the latest coursier. Was this a good upgrade to the new version API?

Reply:
What problem are you running into with these changes?

Reply:
We don't know yet, we have bisected the commit history and gotten to these changes. I think the upgrade is somehow affecting the automatic SemanticDB resolution.